html-css/Task3/index.html

```diff
@@ -0,0 +1,186 @@
+                                non, comm lorem ipin <a href="#" class="orange">read more ...</a></span>
+                        </div>
+                        <div class="article__image article__image_left">
+                                <img src="images/Content3.png" >
+                        </div>
+                    </article>
+                    <article class="article">
+                        <div class="article__textblock article__textblock_right">
+                            <span class="article__title">Nunc dignissim</span>
+                            <span class="article__text">Lorem ipsum dolor sit amet, consec-
+                                tetuer adipiscing elit. Donec odio. Quis-
+                                que volutpat matti <a href="#" class="orange">read more ...</a></span>
+                        </div>
+                        <div class="article__image article__image_right">
+                                <img src="images/Content4.png" >
+                        </div>
+                    </article>
+                </section>
+            </div>     
+        </main> 
+
+        <footer class="page__footer footer">
+            <section class="footer__header">
+                <article class="contact">
+                    <div class="contact__image">
+                        <img src="images/Paper plane.png" >
+                    </div>  
+                    <div class="contact__textblock">
+                        <span class="contact__text">faceboom.com/sitename
+                        @sitename</span>
+                    </div> 
+                </article>
+
+                <article class="contact">
+                    <div class="contact__image">
+                        <img src="images/testimonial.png" >
+                    </div> 
+                    <div class="contact__textblock">
+                        <span class="contact__text">“ Donec consectetuer vulputate se
+                            m tristiq donec odio com ”</span>
+                    </div>
+      
+                </article>
+                <article class="contact">
+                    <div class="contact__image">
+                        <img src="images/User.png" >
+                    </div> 
+                    <div class="contact__textblock">
+                        <span class="contact__text"><a class="orange" href="mailto:info@sitename.com">Email : info@sitename.com</a>
+                        <a class="orange" href="callto:7777777777">Call Us : (000) 777 777 7777</a></span>
+                    </div>
+                  
+                </article>
+            </section>
+            <div class="footer__line"></div>
+            <div class="footer__body">
+                <div class="subblock">
+                    <span class="subblock__title">Vivamus vesinqu</span>
+                    <form action="#">
+                        <input type="email" value="your email id here...">
```

Comment:
Is it possible to use placeholder instead of value?
Otherwise when I try to enter my email I have to remove the existing value (which is not user friedly).